example/example-input.json

```diff
@@ -27,17 +28,17 @@
     "pages": [{
         "label": "Title",
         "file": "file53.tiff",
-        "iiif-server": "https://image-server.library.nd.edu:8182/iif/2/2018_example_001/"
+        "iiif-server": "https://image-server.library.nd.edu:8182/iif/2"
```

Comment:
It seems to me if this is going to be the same on every field it should be a parameter at the top.

Reply:
Yeah I was kind of wondering how you'd want that done, but it definitely doesn't NEED to be duplicated.

Reply:
It is- see line 4

Reply:
I believe Jon is saying that lines 31, 36, and 41 should be moved up to the header section as well since they are all the same.